pom.xml

```diff
@@ -131,6 +125,21 @@
                 <artifactId>tyrus-standalone-client</artifactId>
                 <version>${websocket-tyrus.version}</version>
             </dependency>
+            <dependency>
```

Comment:
can you elaborate on these logback/slf4j changes ? what incompatibilty ? an Exception ?